components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -522,7 +523,17 @@ private long getTotalUsers(String domainName) throws CharonException {
             try {
                 AbstractUserStoreManager secondaryUserStoreManager = (AbstractUserStoreManager) carbonUM
                         .getSecondaryUserStoreManager(domainName);
-                totalUsers = secondaryUserStoreManager.countUsersWithClaims("http://wso2.org/claims/username", "*");
+                totalUsers = secondaryUserStoreManager.countUsersWithClaims(USERNAME_CLAIM, "*");
+            } catch (org.wso2.carbon.user.core.UserStoreException e) {
+                throw new CharonException("Error while getting total user count in domain: " + domainName);
+            }
+        } else if (StringUtils.isNotBlank(domainName)) {
+            try {
+                AbstractUserStoreManager secondaryUserStoreManager = (AbstractUserStoreManager) carbonUM
```

Comment:
instead of having two local variable inside the if statement and else-if statement let's assign the variable in the beginning, and re-use the same variable in both conditions.